core/src/test/java/io/questdb/test/griffin/SqlCompilerImplTest.java

```diff
@@ -4599,6 +4618,21 @@ public void testNonEqualityJoinCondition() throws Exception {
         });
     }
 
+    @Test
+    public void testNotCachedRecordCursorFactory() throws SqlException {
+        try (SqlCompilerImpl compilerX = new SqlCompilerImpl(engine) {
```

Comment:
pls use `engine.compile()`

Reply:
but I need to return something that implements the interface, so I need to override the unknown show statement method, that is why I do it that way.

Reply:
sorry, my bad :(